backend/measurement-service/src/main/resources/measurement-service-persister.yml

```
@@ -0,0 +1,116 @@
+
+            - jsonPath: $.measurements.*.wfStatus
+
+            - jsonPath: $.measurements.*.additionalDetails
+              type: JSON
+              dbType: JSONB
+
+            - jsonPath: $.measurements.*.auditDetails.lastModifiedTime
+
+            - jsonPath: $.measurements.*.auditDetails.lastModifiedBy
+
+            - jsonPath: $.measurements.*.id
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)